content/docs/hooks/use-slider.mdx

```diff
@@ -0,0 +1,161 @@
+| `getInnerTrackProps` | `PropGetter`                            | A function that takes slider inner track style props and handles changes in the slider's inner track style.      |
+| `getThumbProps`      | `PropGetter`                            | A function that takes slider thumb props and handles changes for the slider's thumb.                             |
+| `getMarkerProps`     | `RequiredPropGetter<{ value: number }>` | A function that takes the value of the marker and handles changes for the slider's marker positioning and style. |
+| `getInputProps`      | `PropGetter`                            | A function to get the props of the input field.                                                                  |
+
+## Usage
+
+```jsx
+function Example() {
+  const CustomComponent = (props) => {
+    const { stepByNumber, stepToNumber, ...sliderProps } = props
+    const {
+      state,
+      actions,
+      getInnerTrackProps,
+      getInputProps,
+      getMarkerProps,
+      getRootProps,
+      getThumbProps,
+      getTrackProps,
+    } = useSlider(sliderProps)
+
+    const firstMarker = getMarkerProps({ value: 25 })
+    const secondMarker = getMarkerProps({ value: 50 })
+    const thirdMarker = getMarkerProps({ value: 75 })
+
+    const { onKeyDown: onThumbKeyDown, ...thumbProps } = getThumbProps()
+
+    return (
+      <Box h={['160px', '120px']}>
+        <Flex flexDir='row' justifyContent='space-between'>
+          <Text>
+            Use
+            <Kbd>&#8592;</Kbd> or <Kbd>&#8594;</Kbd>
+            <Text>to step by {stepByNumber}</Text>
+          </Text>
+          <Button
+            size='sm'
+            colorScheme='teal'
+            onClick={() => actions.stepTo(stepToNumber)}
+          >
+            Step to {stepToNumber}
+          </Button>
+          <Button size='sm' colorScheme='red' onClick={() => actions.reset()}>
+            Reset to {props.defaultValue}
+          </Button>
+        </Flex>
+        <chakra.div
+          mt={2}
+          cursor='pointer'
+          w={{ base: '96%', lg: '98%' }}
+          ml={{ base: '2%', lg: '1%' }}
+          {...getRootProps()}
+        >
+          <input {...getInputProps()} hidden />
+          <Badge
+            colorScheme='red'
+            ml='-20px'
+            mt='25px'
+            fontSize='sm'
+            {...firstMarker}
+          >
+            25%
+          </Badge>
+          <Badge
+            colorScheme='red'
+            ml='-20px'
+            mt='25px'
+            fontSize='sm'
+            {...secondMarker}
+          >
+            50%
+          </Badge>
+          <Badge
+            colorScheme='red'
+            ml='-20px'
+            mt='25px'
+            fontSize='sm'
+            {...thirdMarker}
+          >
+            75%
+          </Badge>
```

Comment:
Since these badges take a big portion of the screen, I would put them in a map and do something like:
badges.map(...)

This way we can make the code shorter. 

One thing you can also look into is use CodeSandbox and seperate things into files, that way it won't be so overwhelming for the user. Please check the theming tabs for the Button component to see an example.